tests/server/api/v1/test_records.py

```diff
@@ -145,3 +158,69 @@ def test_create_record_response_with_nonexistent_record_id(client: TestClient, d
 
     assert response.status_code == 404
     assert db.query(Response).count() == 0
+
+
+def test_list_record_responses(client: TestClient, db: Session, admin: User, admin_auth_header: dict):
+    record = RecordFactory.create()
+    response_a = ResponseFactory.create(record=record)
+    response_b = ResponseFactory.create(record=record)
+
+    response = client.get(f"/api/v1/records/{record.id}/responses", headers=admin_auth_header)
+
+    assert response.status_code == 200
+    assert response.json() == {
+        "items": [
+            {
+                "id": str(response_a.id),
+                "inserted_at": response_a.inserted_at.isoformat(),
+                "updated_at": response_a.updated_at.isoformat(),
+                "values": response_a.values,
+                "user_id": str(response_a.user_id),
+            },
+            {
+                "id": str(response_b.id),
+                "inserted_at": response_b.inserted_at.isoformat(),
+                "updated_at": response_b.updated_at.isoformat(),
+                "values": response_b.values,
+                "user_id": str(response_b.user.id),
+            },
+        ]
+    }
+
+
+def test_list_record_responses_as_annotator(client: TestClient, db: Session, admin: User, admin_auth_header: dict):
+    record = RecordFactory.create()
+    annotator = AnnotatorFactory.create(workspaces=[record.dataset.workspace])
+    response_a = ResponseFactory.create(record=record, user=annotator)
+    ResponseFactory.create(record=record, user=admin)
+    ResponseFactory.create(record=record)
+
+    response = client.get(f"/api/v1/records/{record.id}/responses", headers={API_KEY_HEADER_NAME: annotator.api_key})
+
+    assert response.status_code == 200
+    assert response.json() == {
+        "items": [
+            {
+                "id": str(response_a.id),
+                "inserted_at": response_a.inserted_at.isoformat(),
+                "updated_at": response_a.updated_at.isoformat(),
+                "values": response_a.values,
+                "user_id": str(response_a.user_id),
+            }
+        ]
+    }
+
+
+def test_list_record_responses_as_annotator_from_different_workspace(client: TestClient, db: Session):
+    record = RecordFactory.create()
+    annotator = AnnotatorFactory.create(workspaces=[WorkspaceFactory.build()])
+
+    response = client.get(f"/api/v1/records/{record.id}/responses", headers={API_KEY_HEADER_NAME: annotator.api_key})
+
+    assert response.status_code == 403
+
+
+def test_list_record_responses_with_nonexistent_record_id(client: TestClient, db: Session, admin_auth_header: dict):
+    response = client.get(f"/api/v1/records/{uuid4()}/responses", headers=admin_auth_header)
```

Comment:
It's fine like this but I usually create something on the database to be sure that we are not returning anything:

```suggestion
    record = RecordFactory.create()
    ResponseFactory.create(record=record)

    response = client.get(f"/api/v1/records/{uuid4()}/responses", headers=admin_auth_header)
```